packages/driver/src/cypress/runner.ts

```diff
@@ -28,6 +25,12 @@ const RUNNABLE_PROPS = '_testConfig id order title _titlePath root hookName hook
 const debug = debugFn('cypress:driver:runner')
 const debugErrors = debugFn('cypress:driver:errors')
 
+const duration = (before: Date, after: Date) => {
+  // TypeScript doesn't allow subtraction between `Date`s.
+  // @ts-ignore
+  return before - after
```

Comment:
I always forget this is possible, I end up doing `Number(date)` every time

Reply:
I removed the comment and changed it to `Number(before) - Number(after)`. It's much cleaner. Thanks!